resource/stopwords/ja.txt

```diff
@@ -0,0 +1,111 @@
+あ
```

Comment:
Japanese isn't supported, cf. https://github.com/dataiku/dss-plugin-nlp-visualization/blob/9572eb5e0540614f52cb739ae114022c7ffcbd6e/python-lib/language_dict.py#L67

Reply:
Agreed, but this issue will soon be solved (cf https://github.com/WorksApplications/SudachiPy/issues/148)
So I included this file in preparation of a next version which will add Japanase